autoPyTorch/pipeline/components/training/trainer/base_trainer.py

```diff
@@ -119,10 +119,11 @@ def add_performance(self,
         self.performance_tracker['val_metrics'][epoch] = val_metrics
         self.performance_tracker['test_metrics'][epoch] = test_metrics
 
-    def get_best_epoch(self, loss_type: str = 'val_loss') -> int:
-        # If we compute validation scores, prefer the performance
+    def get_best_epoch(self, dataset: str = 'val') -> int:
```

Comment:
`split_type` is more precise?
```suggestion
    def get_best_epoch(self, split_type: str = 'val') -> int:
```